### PR TITLE
use custom appstore id config param

### PR DESCRIPTION
**Pull request by bromiesTM**

### CODE REVIEW

File: core/templates/layout.public.php [hunk 1]
```diff
@@ -16,7 +16,7 @@
 	</title>
 	<meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
 	<?php if ($theme->getiTunesAppId() !== '') { ?>
```
Comment (fracado) [inline core/templates/layout.public.php:18]: Shouldn't we replace it in the condition too? Maybe something like this?
```
<?php $iosAppid = \OC::$server->get(\OC\SystemConfig::class)->getValue("ionos_customclient_ios_appid"); ?>
<?php if ($iosAppid !== '') { ?>
<meta name="apple-itunes-app" content="app-id=<?php p($iosAppid); ?>">
<?php } ?>
```

File: core/templates/layout.user.php [hunk 2]
```diff
@@ -34,7 +34,7 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
 		<?php if ($theme->getiTunesAppId() !== '') { ?>
```
Comment (fracado) [inline core/templates/layout.user.php:36]: see similar comment for `layout.public.php`.

File: core/templates/layout.public.php (hunk 1 above)
Comment (bromiesTM) [reply core/templates/layout.public.php]: [Done](https://github.com/IONOS-Productivity/nc-ionos-theme/pull/11/files)

**Review by fracado — APPROVED** [review]
review OK

**Pull request merged**